src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -240,28 +240,18 @@ private void registerCheck(AbstractCheck check) throws CheckstyleException {
      * @throws CheckstyleException if Check is misconfigured
      */
     private void registerCheck(int tokenId, AbstractCheck check) throws CheckstyleException {
-        registerCheck(TokenUtil.getTokenName(tokenId), check);
-    }
-
-    /**
-     * Register a check for a specified token name.
-     *
-     * @param token the name of the token
-     * @param check the check to register
-     * @throws CheckstyleException if Check is misconfigured
-     */
-    private void registerCheck(String token, AbstractCheck check) throws CheckstyleException {
         if (check.isCommentNodesRequired()) {
-            tokenToCommentChecks.computeIfAbsent(token, empty -> new HashSet<>()).add(check);
+            tokenToCommentChecks.computeIfAbsent(tokenId, empty -> new HashSet<>()).add(check);
         }
-        else if (TokenUtil.isCommentType(token)) {
```

Comment:
You changed `TokenUtil.isCommentType(String)` to `TokenUtil.isCommentType(int)`. Pitest is not happy.

Reply:
Pitest is appeased. Long live pitest.